packaging/installer/kickstart.sh

```diff
@@ -411,18 +411,23 @@ cleanup() {
 
 deferred_warnings() {
   if [ -n "${NETDATA_WARNINGS}" ]; then
+    troubleshoot_guide_url="https://learn.netdata.cloud/docs/installing/installation-troubleshoot-actions"
```

Comment:
This should be a global variable defined up above with the other URLs.

Reply:
Ok